### ISSUE TITLE
Surgical Apron doesn't block bloodsplatter from surgery

### ISSUE THREAD

**Issue by ArbyTheDragon** [Eclipse-Station/NEV-Northern-Light#348]
What I did: Be doctor. Need to remove appendix from person. Wore surgical apron because I'm a good follower of SOP. Did Surgery.

What happened: There's blood on my apron and gloves. That's normal. There's also blood all over my uniform. That's not normal.
_"You got blood on my suit!"_
![image](https://user-images.githubusercontent.com/15035655/87267217-a3c0ab80-c48c-11ea-8cbc-7a8c8802d0aa.png)

I thought the whole point of the surgery apron is to prevent you from getting your uniform filthy.

**Comment by ShadzFen** [comment]
This might be intentional. I did surgery on a monkey about 5 times and didn't get any blood on my suit or the apron. I only managed to get blood on myself when I attacked the monkey with a surgical saw and blood splattered across the room.

![image](https://user-images.githubusercontent.com/45645502/87340877-f95a8e00-c548-11ea-99ec-b31e36049756.png) 

Looking at the code, the apron covers the torso and lower body, but not the arms, while jumpsuits do cover arms. If blood sprays on your arms, your jumpsuit is `technically` dirty with blood, and needs to be washed. It's pretty clear from the sprite that the apron doesn't cover the arms.

![image](https://user-images.githubusercontent.com/45645502/87340912-07a8aa00-c549-11ea-9961-27d6b5610611.png)

Not sure if intentional or not. If it is intentional, it could make the apron pointless, unless it has some weird stat that makes it so there's a lower chance of getting blood on you. I don't know.
